PadoruLogger/Debug.cs

```diff
@@ -99,87 +97,26 @@ public static void AddOutput(IDebugOutput output)
             outputs.Add(output);
         }
 
-        public static void Log(object message = null)
-        {
-            Log(message, DEFAULT_CHANNEL_NAME, null);
-        }
-
-        public static void Log(object message, string channel)
-        {
-            Log(message, channel, null);
-        }
-
-        public static void Log(object message, object context)
-        {
-            Log(message, DEFAULT_CHANNEL_NAME, context);
-        }
-
-        public static void Log(object message, string channel, object context)
+        public static void Log(object message, string channel, object context=null)
         {
             InternalLog(LogType.Info, message, channel, context);
         }
 
-
-        public static void LogWarning(object message = null)
-        {
-            LogWarning(message, DEFAULT_CHANNEL_NAME, null);
-        }
-
-        public static void LogWarning(object message, string channel)
-        {
-            LogWarning(message, channel, null);
-        }
-
-        public static void LogWarning(object message, object context)
-        {
-            LogWarning(message, DEFAULT_CHANNEL_NAME, context);
-        }
-
-        public static void LogWarning(object message, string channel, object context)
+        public static void LogWarning(object message, string channel, object context=null)
         {
             InternalLog(LogType.Warning, message, channel, context);
         }
 
-        public static void LogError(object message = null)
-        {
-            LogError(message, DEFAULT_CHANNEL_NAME, null);
-        }
-
-        public static void LogError(object message, string channel)
-        {
-            LogError(message, channel, null);
-        }
-
-        public static void LogError(object message, object context)
-        {
-            LogError(message, DEFAULT_CHANNEL_NAME, context);
-        }
-
-        public static void LogError(object message, string channel, object context)
+        public static void LogError(object message, string channel, object context=null)
         {
             InternalLog(LogType.Error, message, channel, context);
         }
 
-        public static void LogException(Exception e)
```

Comment:
We should keep this overload, no reason to sent null as the messageHeader if we do not want one